src/main/java/com/launchacademy/reviews/repositories/LanguageRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.launchacademy.reviews.repositories;
+
+import com.launchacademy.reviews.models.Language;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface LanguageRepository extends CrudRepository<Language, Integer> {
+  public Language findByLanguage(String language);
```

Comment:
Ideally, I prefer to not have column names and entity/table names overlap (e.g., I'd like the `languages` table to have a `name` column, instead of a `language` column). Not blocking, but worth mentioning. We can already see here how it starts to confuse our naming as time goes on :)